post_process_transforms/UTK_ir_etds_post_process.xsl

```diff
@@ -96,6 +97,22 @@
   </xsl:template>
 
   <!--
+    based on the value of mods:extension/etd:degree/etd:level, serialize the correct
+    URI for mods:genre[@authority='COAR'].
+  -->
```

Comment:
Right now, this just copies and copies and copies every time it's edited.

Am I nuts, or couldn't we just do this after 102 and before 103:

<xsl:template match="/mods:mods/mods:genre[@authority='coar']"/>

Reply:
That should work - let me gin up a test. Thanks for the suggestion.

Reply:
I added that and tested a half a dozen times and it seemed to work for me.

Reply:
@markpbaggett my concern: a `template match='mods:genre[@authority='coar']` may drop pre-existing `mods:genre[@authority='coar']` on UPDATEs.

Reply:
I know we talked about this, but adding this to the record for history's sake:

According to this, templates always match in the source XML.  For that reason, I don't think we should have an issue with the new template firing after your's.

https://stackoverflow.com/questions/1531664/in-what-order-do-templates-in-an-xslt-document-execute-and-do-they-match-on-the/1542734

Reply:
That's perfect -- thanks for pointing that out!